cinema/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=100)
+    description = serializers.CharField(max_length=255)
```

Comment:
The `description` field in the `MovieSerializer` has a `max_length` of 255, which does not match the `TextField` in the `Movie` model that does not have a `max_length` constraint. Consider using `serializers.CharField` without `max_length` or using `serializers.TextField` to align with the model definition.